src/libraries/Microsoft.Extensions.Configuration.Binder/tests/SourceGenerationTests/Baselines/ConfigurationBinder/Bind.generated.txt

```diff
@@ -45,7 +45,7 @@ namespace Microsoft.Extensions.Configuration.Binder.SourceGeneration
             }
 
             var typedObj = (Program.MyClass)instance;
-            BindCore(configuration, ref typedObj, binderOptions: null);
+            BindCore(configuration, ref typedObj, false, binderOptions: null);
```

Comment:
```suggestion
            BindCore(configuration, ref typedObj, defaultValueIfNotFound: false, binderOptions: null);
```

Reply:
OK; originally I was assuming we wanted terse generated code, but I see that we use `binderOptions: null`

Reply:
We do want terse code, but generally prioritize readability.